contrib/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+# Third-party libraries may have substandard code.
```

Comment:
Add S2 to the list of third party libraries that are being used: https://github.com/ClickHouse/ClickHouse/blob/ca2171695f47551edcd9ee906c9a81843a011650/docs/en/development/contrib.md